tests/unittests/evaluation_tests.py

```
@@ -1977,6 +2006,61 @@ def test_evaluate_polylines_coco(self):
+                label="pose",
+                points=[(0, 0), (0.7, 1)],
+            )
+        ]
+
+        default = foui.compute_ious(preds, gts)[0, 0]
+        strict = foui.compute_ious(preds, gts, keypoint_sigmas=[1, 0.05])[0, 0]
+
+        self.assertGreater(default, 0.9)
+        self.assertLess(strict, 0.9)
+
+        with self.assertRaises(ValueError):
```

Ruff (0.15.17)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)